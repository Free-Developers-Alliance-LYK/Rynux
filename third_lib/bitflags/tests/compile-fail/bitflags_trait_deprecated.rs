#![deny(warnings)]

pub use bitflags::BitFlags;

fn main() {}
