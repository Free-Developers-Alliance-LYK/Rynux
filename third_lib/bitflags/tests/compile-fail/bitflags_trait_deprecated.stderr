error: use of deprecated trait `bitflags::BitFlags`: use the `Flags` trait instead
 --> tests/compile-fail/bitflags_trait_deprecated.rs:3:19
  |
3 | pub use bitflags::BitFlags;
  |                   ^^^^^^^^
  |
note: the lint level is defined here
 --> tests/compile-fail/bitflags_trait_deprecated.rs:1:9
  |
1 | #![deny(warnings)]
  |         ^^^^^^^^
  = note: `#[deny(deprecated)]` implied by `#[deny(warnings)]`
